presto-manager-common/src/main/java/com/teradata/prestomanager/common/StopType.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.teradata.prestomanager.agent;
```

Comment:
Yes, will do that next time encountering this situation. Already have too many other ones on top of it.